dataflow/manual/content.tex

```diff
@@ -875,11 +875,6 @@ \subsection{Formal Definition of the Control-Flow Graph}
 \end{workinprogress}
 
 \begin{workinprogress}
-There is a \code{CaseNode}. How is the default case of a switch
-handled?
-\end{workinprogress}
-
-\begin{workinprogress}
 There is a \code{StringConcatenateAssignmentNode}.
 What about other compound assignments? Are they desugared?
```

Comment:
Now that you looked at unary assignments and other CFG nodes, can you, in a separate PR, address this `workinprogress` about compound assignments?